packages/visualizations/src/components/Map/Combinations/Tooltip.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+    export let label = '';
+    let containerElement: HTMLDivElement;
+    let x: number;
+    let y: number;
+    let hovered = false;
+
+    function showTooltip() {
```

Comment:
Why not tippy? I supposed the tooltip we have on KPIs looked similar to what we expect here, and we'd have visual consistency and maybe some goodies we don't think of (accessibility like showing on focus or stuff like that)